tests/test_utils.py

```diff
@@ -279,13 +267,9 @@ def test_save_load_state_dict_train_after_old_model(self):
             model.load_state_dict(load(file))
 
         # Compare the new model weights and biases. they should now be in sync
-        (
-            new_model_weights,
-            new_model_biases,
-            new_tile_weights,
-            new_tile_biases,
-            _,
-        ) = self.get_layer_and_tile_weights(model)
+        (new_model_weights, new_model_biases, new_tile_weights, new_tile_biases, _) = (
+            self.get_layer_and_tile_weights(model)
+        )
```

Comment:
Do you use a different version of black so that you get this formatting changes? There is a specific version set in the requirements. please update that or use the same version

Reply:
Yes I was using a newer version. I changed it back.